index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Odin Library</title>
+    <link rel="stylesheet" href="/assets/css/style.css">
```

Comment:
add a dot `(.)` at the beginning of the path.
Final version `./assets/css/style.css`